test/test_array_ops.cpp

```diff
@@ -304,6 +304,35 @@ TEST(ArrayOps, QuantizeV2i8) {
   opexecuter.RunTest();
 }  // end of test op QuantizeV2i8
 
+// Test op: QuantizeV2
+// Quantizes a tensor from float to i8. Also tests min-max output
+// TODO: enable this test when min-max output generation is supported
+TEST(ArrayOps, DISABLED_QuantizeV2i8minmax) {
+  Scope root = Scope::NewRootScope();
+  int dim1 = 2;
+  int dim2 = 3;
+
+  Tensor A(DT_FLOAT, TensorShape({dim1, dim2}));
+  AssignInputValues<float>(A, {-0.9, -1.3, 2.6, 3.5, 4.2, 5.0});
```

Comment:
@mingshan-wang I think the specific data is required and he is assigning values from vector using AssingInputValues.

Reply:
I specifically use these values because I want a known mix of negative, positive and numbers that are `x.5`, because they are important to test the rounding mode. With random assignment I cannot have that control

Reply:
I see. Cool, thanks for the explanation.